### PR TITLE
[DNM] Always publish scans.

### PR DESCRIPTION
**Pull request by runningcode**
Do not merge, this is just to test the CI pipeline.

### CODE REVIEW

File: common-custom-user-data-gradle-plugin/settings.gradle [hunk 1]
```diff
@@ -4,3 +4,11 @@ plugins {
...
 
 rootProject.name = 'common-custom-user-data-gradle-plugin'
+
+gradleEnterprise {
+    buildScan {
+        termsOfServiceUrl = "https://gradle.com/terms-of-service"
+        termsOfServiceAgree = "yes"
+        publishAlways()
+    }
+}
```
Comment (etiennestuder) [inline common-custom-user-data-gradle-plugin/settings.gradle:14]: We cannot check this in @runningcode . People need to have a choice to agree or not. They can simply run the build with `--scan`.

Comment (runningcode) [reply common-custom-user-data-gradle-plugin/settings.gradle:14]: Ah sorry my goal here was to try to trigger the CI pipeline by creating a PR. I've updated the description to make that clear.

**Comment by etiennestuder** [comment]
@runningcode do we still need this PR?

**Comment by runningcode** [comment]
Not needed. Thanks

**Pull request closed without merging**